cpp/velox/shuffle/VeloxShuffleWriter.cc

```diff
@@ -1299,7 +1362,17 @@ arrow::Status VeloxShuffleWriter::splitFixedWidthValueBuffer(const velox::RowVec
       complexTypeData_[partitionId] = nullptr;
     }
 
-    return makeRecordBatch(numRows, allBuffers, writeSchema(), pool_.get());
+    return makeRecordBatch(numRows, allBuffers);
+  }
+
+  std::shared_ptr<arrow::RecordBatch> VeloxShuffleWriter::makeRecordBatch(
+      uint32_t numRows, const std::vector<std::shared_ptr<arrow::Buffer>>& buffers) {
+    // auto isTinyBatch = numRows <= options_.batch_compress_threshold;
```

Comment:
Why comment out this line?

Reply:
For uncompression codec type, we won't copy the data, it should use the writeSchema(), not the compressWriteSchema, isTinyBatch will destroy this check.